<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setupDialog</class>
 <widget class="QDialog" name="setupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>210</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="serverBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string notr="true">Server connection</string>
   </property>
   <widget class="QLineEdit" name="serverIPEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverPortEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ipLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Server IP&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;Most interesting things will be hidden in the enclave
Which makes the enclave the first priority target
</string>
    </property>
   </widget>
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Server port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testConn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Test connection</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="couponBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>511</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string notr="true">Enter coupon</string>
   </property>
   <widget class="QTextEdit" name="couponText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>431</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="applyBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>51</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelBalance">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Balance is ...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="folderBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Library location</string>
   </property>
   <widget class="QLineEdit" name="folderEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="folderChooseButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose folder and init user</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>setupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>setupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>testConn</sender>
   <signal>clicked()</signal>
   <receiver>setupDialog</receiver>
   <slot>testConnSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>folderChooseButton</sender>
   <signal>clicked()</signal>
   <receiver>setupDialog</receiver>
   <slot>chooseFolderSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>820</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyBtn</sender>
   <signal>clicked()</signal>
   <receiver>setupDialog</receiver>
   <slot>applyCouponSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>845</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>testConnSlot()</slot>
  <slot>applyCouponSlot()</slot>
  <slot>chooseFolderSlot()</slot>
 </slots>
</ui>
